### PR TITLE
explain-overview: update a description to consider tiflash mpp cases

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
explain-overview: add a description to include tiflash mpp cases

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: explain-overview.md [hunk 1]
```diff
@@ -94,7 +94,9 @@ In the `WHERE`/`HAVING`/`ON` conditions, the TiDB optimizer analyzes the result
 
 Currently, calculation tasks of TiDB can be divided into two categories: cop tasks and root tasks. A `cop[tikv]` task indicates that the operator is performed inside the TiKV coprocessor. A `root` task indicates that it will be completed inside of TiDB.
 
-One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. The Coprocessor in TiKV supports most of the built-in SQL functions (including the aggregate functions and the scalar functions), SQL `LIMIT` operations, index scans, and table scans. However, all `Join` operations can only be performed as root tasks in TiDB.
+One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. The Coprocessor in TiKV supports most of the built-in SQL functions (including the aggregate functions and the scalar functions), SQL `LIMIT` operations, index scans, and table scans.
+
+ not pushed down to TiKV and are only performed as root tasks in TiDB. However, `Join` operations are also performed in the [TiFlash MPP](/tiflash/use-tiflash-mpp-mode.md) mode in some cases and these `Join` operations are pushed down to TiFlash nodes.
```
Comment (TomShawn) [inline explain-overview.md:99]:
```suggestion
`Join` operations are not pushed down to TiKV and are only performed as root tasks in TiDB. However, for `Join` operations performed in the [TiFlash MPP](/tiflash/use-tiflash-mpp-mode.md) mode in some cases, these `Join` operations are pushed down to TiFlash nodes.
```

File: explain-overview.md [hunk 2]
```diff
@@ -94,7 +94,9 @@ In the `WHERE`/`HAVING`/`ON` conditions, the TiDB optimizer analyzes the result
 
 Currently, calculation tasks of TiDB can be divided into two categories: cop tasks and root tasks. A `cop[tikv]` task indicates that the operator is performed inside the TiKV coprocessor. A `root` task indicates that it will be completed inside of TiDB.
 
-One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. The Coprocessor in TiKV supports most of the built-in SQL functions (including the aggregate functions and the scalar functions), SQL `LIMIT` operations, index scans, and table scans. However, all `Join` operations can only be performed as root tasks in TiDB.
+One of the goals of SQL optimization is to push the calculation down to TiKV as much as possible. The Coprocessor in TiKV supports most of the built-in SQL functions (including the aggregate functions and the scalar functions), SQL `LIMIT` operations, index scans, and table scans.
+
+`Join` operations are not pushed down to TiKV and are only performed as root tasks in TiDB. However, for `Join` operations performed in the [TiFlash MPP](/tiflash/use-tiflash-mpp-mode.md) mode in some cases, these `Join` operations are pushed down to TiFlash nodes.
```
Comment (TomShawn) [inline explain-overview.md:99]:
```suggestion
```

**Review by windtalker — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]

**Review by samhld — APPROVED** [review]
Yeah, that works!

**Pull request merged**